chromadb/api/models/Collection.py

```diff
@@ -578,6 +615,12 @@ def _validate_embedding_set(
             valid_uris,
         )
 
+    @staticmethod
+    def _normalize_embeddings(embeddings: Embeddings) -> Embeddings:
```

Comment:
Isn't the input type here wrong? Embeddings could be np.ndarray

Reply:
updated to use both Embeddings and np.ndarray